spec/models/state_machines/achievement_state_machine_spec.rb

```diff
@@ -44,6 +44,20 @@
           .to raise_error(Statesman::TransitionFailedError)
       end
     end
+
+    it "can transition from complete to other states" do
+      %i[rejected].each do |allowed_state|
+        expect { create(:completed_achievement).state_machine.transition_to!(allowed_state) }
+          .not_to raise_error
+      end
+    end
+
+    it "can transition from rejected to other states" do
```

Comment:
I believe they are the right way round, this one is taking a rejected achievement and checking it can be convereted to completed or drafted. The one on line 48 is checking a completed achievement can be converted to rejected. I can tweak the wording if its confusing, but was just mirroring what we had already.

Reply:
You are quite right, my mistake! No need to change it, just bad reading comprehension on my part.